### PR TITLE
Fixing: Add pref xpinstall.signatures.dev-root to docs

### PR DESCRIPTION
**Pull request by championshuttler**
Fixing #4428

### CODE REVIEW

**Comment by championshuttler** [comment]
@bobsilverberg , sure I will make changes soon :)

**Comment by championshuttler** [comment]
@bobsilverberg , as @muffinresearch is suggesting an easier way to do the same , should we add this procedure or go with user.js ?

**Comment by bobsilverberg** [comment]
Ah, I had no idea it could be done like that. Thanks @muffinresearch!

@championshuttler Please go with the technique described by @muffinresearch.

**Comment by bobsilverberg** [comment]
Also, please add the documentation to https://github.com/mozilla/addons-frontend/blob/master/docs/moz-addon-manager.md#install-add-ons-for-development, rather than creating a new document.

**Comment by championshuttler** [comment]
@bobsilverberg I made changes , please give it a look :)

**Comment by championshuttler** [comment]
@bobsilverberg please check, is it looking good now ;)

File: docs/moz-addon-manager.md [hunk 1]
```diff
@@ -30,6 +30,10 @@ Go to `about:config` in Firefox and set this property to `true`:
 xpinstall.signatures.dev-root
 ```
 
+## Adding pref `xpinstall.signatures.dev-root`
+
+This pref is not exist by default, so to add this pref, right click in `about:config` in Firefox , select `new` and then add `xpinstall.signatures.dev-root` as `Boolean`.
```
[the remark this answers is not in the document]
Comment (championshuttler) [reply docs/moz-addon-manager.md]: @muffinresearch please take a look i removed the header as bob siggested and changed the line also ;)

File: docs/moz-addon-manager.md [hunk 2]
```diff
@@ -30,6 +30,10 @@ Go to `about:config` in Firefox and set this property to `true`:
 xpinstall.signatures.dev-root
 ```
 
+If you do not see an entry for `xpinstall.signatures.dev-root` in `about:config` you will need to add one by doing the following: 
+
+Right click in `about:config` in Firefox , select `new` and then add `xpinstall.signatures.dev-root` as `Boolean`.
```
Comment (bobsilverberg) [inline docs/moz-addon-manager.md]: I think you can remove the `in Firefox` here, as we've already said above "you will need to tell Firefox to honor their signing certificates".

**Comment by championshuttler** [comment]
@bobsilverberg done ;)

**Pull request merged**